cfme/tests/perf/workloads/test_provisioning.py

```diff
@@ -77,9 +77,7 @@ def test_provisioning(appliance, request, scenario):
     Memory Monitor creates graphs and summary at the end of each scenario.
 
     Polarion:
-        assignee: pvala
-        casecomponent: Rest
-        caseimportance: high
+        assignee: rhcf3_machine
```

Comment:
Why are you un-assigning this test case?

Reply:
It is a performance-related test case, which I had mistakenly assigned to myself, so assigning it back rhcf3_machine.